src/librustc_mir/borrow_check.rs

```diff
@@ -975,10 +975,19 @@ impl<'c, 'b, 'a: 'b+'c, 'gcx, 'tcx: 'a> MirBorrowckCtxt<'c, 'b, 'a, 'gcx, 'tcx>
         err.emit();
     }
 
-    fn report_illegal_mutation_of_borrowed(&mut self, _: Context, (lvalue, span): (&Lvalue, Span)) {
+    fn report_illegal_mutation_of_borrowed(&mut self,
```

Comment:
I think this should be moved to `borrowck_errors` actually, but that could be done in another commit.

Reply:
You mean like @mikhail-m1 has done in #44882 ?

Reply:
Yeah

Reply:
Done here: #44922